tests/Agent/IntegrationTests/UnboundedIntegrationTests/Elasticsearch/ElasticsearchTests.cs

```diff
@@ -39,20 +39,15 @@ protected ElasticsearchTestsBase(TFixture fixture, ITestOutputHelper output, Cli
             _fixture.TestLogger = output;
             _clientType = clientType;
 
-            // TODO: Set high to allow for debugging
-            _fixture.SetTimeout(TimeSpan.FromMinutes(20));
+            _fixture.SetTimeout(TimeSpan.FromMinutes(2));
 
-            _fixture.AddCommand($"ElasticsearchExerciser SetClient {clientType}");
-
-            // Sync operations
-
-            _fixture.AddCommand($"ElasticsearchExerciser Index");
-
-            _fixture.AddCommand($"ElasticsearchExerciser Search");
-
-            _fixture.AddCommand($"ElasticsearchExerciser IndexMany");
+            if (_clientType != ClientType.ElasticClients)
+            {
+                // This lets 7.x clients work with an 8.x server
+                _fixture.SetAdditionalEnvironmentVariable("ELASTIC_CLIENT_APIVERSIONING", "true");
```

Comment:
Another option could be to not evaluate whether the Elastic Calls succeeded for the older clients, and to potentially just log out whether the call succeeded or not.